src/components/CollapsibleNavigationMenu.tsx

```diff
@@ -1,13 +1,13 @@
-import { useEffect, useState } from 'react';
-import styled, { css, type AnyStyledComponent } from 'styled-components';
+import { Root, List, Item, Sub, Link } from '@radix-ui/react-navigation-menu';
 import _ from 'lodash';
+import { useEffect, useState } from 'react';
```

Comment:
nit: react should always be the first import. Is there a way to separate this from third party imports and not have the forced newline?

Reply:
Oh good call! hmmmm Not sure if it can be done with out a newline, but I think it could be ok if we have a section just for "react"?